src/platform/site-wide/sass/modules/_m-campaign-landing-page.scss

```diff
@@ -2,8 +2,12 @@
   width: 40px;
 }
 
+.va-c-blue-line--large {
+  width: 50px;
+}
+
 .va-u-box-shadow--none {
-  box-shadow: none;
+  box-shadow: none !important;
```

Comment:
I guess there's no `box-shadow` style in the design system?

Reply:
There is not ✅